docker/.env.example

```diff
@@ -20,23 +24,6 @@ WREN_UI_VERSION=0.8.0
 IBIS_SERVER_VERSION=0.5.1
 WREN_BOOTSTRAP_VERSION=0.1.4
 
-# keys
-LLM_PROVIDER=openai
-DOCUMENT_STORE_PROVIDER=qdrant
-# CHANGE THIS TO YOUR OPENAI API KEY
-OPENAI_API_KEY=sk-1234567890
-OPENAI_API_BASE=https://api.openai.com/v1
```

Comment:
Can you double-check whether these environment variables were in use or not?
OPENAI_API_BASE is used in wren-ai-service/docker/docker-compose.yaml file